src/reanimated2/mock.js

```diff
@@ -25,6 +25,9 @@ const ReanimatedV2 = {
     cb && setTimeout(() => cb(true), 0);
     return 0;
   },
+  withDelay: () => {
+    return 0;
+  },
```

Comment:
The definition of `withDelay` is `export function withDelay(delayMs, _nextAnimation)` so I think that it should be:
```js
withDelay: (delayMs, cb) => {
  cb && setTimeout(() => cb(true), delayMs);
  return 0;
},
```

Reply:
We definitely don't want to delay those methods and I think every timeout in this file can be replaced with just `cb && cb(true)`

Reply:
@jakub-gonet thankx

Reply:
@piaskowyk @jakub-gonet 
But the second argument of `withDelay` is an animation object not callback.
In docs, this is used like `withDelay(1000, withTiming(70));`.

I think the `withDelay` mock should be like
```js
  withDelay: (_, animationValue) => {
    return animationValue;
  },
```